ci-test.sh

```diff
@@ -20,11 +20,11 @@ msg "Testing julia-x-y is installed"
 julia-$major -v
 
 msg "Installing specific Julia version"
-echo "yy" | bash jill.sh --version $VERSION
-julia-$VERSION -v
+echo "yy" | bash jill.sh --version $version
+julia-$version -v
 
 msg "Testing if the version is correct"
-[[ $(julia-$VERSION -v) == "julia version $VERSION" ]]
+[[ $(julia-$version -v) == "julia version $version" ]]
```

Comment:
`VERSION` is the variable set inside GitHub Actions, you can leave it that way.

Reply:
oh ok, I was using it to run the install script and it kept creating bad links. sorry about that.

Reply:
removed my ci-test changes.